salt/lax/config/etc-nginx-sitesenabled-bot-lax.conf

```diff
@@ -0,0 +1,66 @@
+# the upstream component nginx needs to connect to
+upstream flask {
+    server unix:///tmp/bot-lax-uwsgi.sock;
+}
+
+{% if salt['elife.cfg']('cfn.outputs.DomainName') %}
+# close connection on those that don't specify a 'host' header
+# http://nginx.org/en/docs/http/request_processing.html
+server {
+    listen 80;
+    listen 443;
+    server_name "";
+    return 444;
+}
+{% endif %}
+
+# configuration of the server
+server {
+    # we always listen on port 8001
+    # in non-dev environments port 8001 is only available to internal traffic
+    {% if salt['elife.cfg']('cfn.outputs.DomainName') %}
+    listen      8001 ssl; # see /etc/nginx/nginx.conf for global ssl settings
+    server_name {{ salt['elife.cfg']('project.project_hostname') }}
+                {{ salt['elife.cfg']('project.full_hostname') }}
+                {{ salt['elife.cfg']('project.int_project_hostname') }}
+                {{ salt['elife.cfg']('project.int_full_hostname') }};
+    {% else %}
+    listen      8001;
+    server_name localhost;
+    {% endif %}
+
+    charset     utf-8;
+
+    access_log /var/log/nginx/bot-lax.access.log combined_with_time;
+    error_log /var/log/nginx/bot-lax.error.log;
+
+    # max upload size
+    # at time of writing (2017-06-28) the largest xml files are:
+    # $ du -h *.xml | sort -n
+    # 660K  elife-25051-v3.xml
+    # 876K  elife-22644-v1.xml
+    # 924K  elife-10056-v2.xml
+    # 976K  elife-18566-v2.xml
+    client_max_body_size 5M;
+
+    location / {
+        uwsgi_pass flask;
+        # drop connection after this many seconds
+        # WARNING: this value *must* be higher than uwsgi's 'harakiri' value (79s) in /srv/lax/uwsgi.ini
+        # at time of writing (2017-06-28) the average scrape time of our largest article was ~40 seconds,
+        # excluding validation and lax round trip
+        uwsgi_read_timeout 80s;
+        # boilerplate, shared with lax uwsgi service
+        include /etc/uwsgi/params;
+
+        # POST/PUT/UPDATE/whatever methods are restricted
```

Comment:
I agree on checking out possible authentication models, but I would avoid the distinction between HTTP methods and just protected all GET/POST/...